aws/resource_aws_securityhub_custom_action_test.go

```diff
@@ -0,0 +1,106 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func testAccAWSSecurityHubCustomAction_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSSecurityHubAccountDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSSecurityHubCustomActionConfig,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSSecurityHubAccountExists("aws_securityhub_account.example"),
+					testAccCheckAWSSecurityHubCustomActionExists("aws_securityhub_custom_action.example"),
+				),
+			},
+			{
+				ResourceName:      "aws_securityhub_custom_action.example",
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
+					"identifier",
+				},
```

Comment:
We should ensure this test passes with this removed, otherwise operators will always be presented with a forces new resource plan immediately after import.

```suggestion
```